make the example work correctly
things are renamed when building the package, and it needs a shebang.

--- a/canvas_demo/canvas.pl
+++ b/canvas_demo/canvas.pl
@@ -1,12 +1,13 @@
+#!/usr/bin/perl
 use strict;
 use Gnome2::Canvas;
 
-require "canvas-arrowhead.pl";
-require "canvas-curve.pl";
-require "canvas-features.pl";
-require "canvas-fifteen.pl";
-require "canvas-primitives.pl";
-require "canvas-rich-text.pl";
+require "canvas-arrowhead.pm";
+require "canvas-curve.pm";
+require "canvas-features.pm";
+require "canvas-fifteen.pm";
+require "canvas-primitives.pm";
+require "canvas-rich-text.pm";
 
 sub create_canvas {
 	my $app = Gtk2::Window->new;
